### PR TITLE
Some search display improvements

### PR DESCRIPTION
**Pull request by not-an-aardvark**

### CODE REVIEW

File: api/services/Reddit.js [hunk 1]
```diff
@@ -38,6 +38,9 @@ var makeRequest = async function (refreshToken, requestType, url, data, rateLimi
   if (left < rateLimitRemainingThreshold && moment().before(resetTime)) {
     throw {statusCode: 504, error: "Rate limited"};
   }
+  if (requestType === 'GET') {
+    url += (url.indexOf('?') === -1 ? '?' : '&') + 'raw_json=1';
+  }
```
Comment (not-an-aardvark) [inline api/services/Reddit.js]: Prevents reddit from replacing `>` with `&gt;`, etc.

Comment (YaManicKill) [inline api/services/Reddit.js]: Is there any reason to only run it on GET requests?

But anyway, really nce, I think this'll fix some of the issues we've had in the past with flairs containing "<"

Comment (YaManicKill) [inline api/services/Reddit.js]: Also, stick a comment in here.

Comment (not-an-aardvark) [inline api/services/Reddit.js]: > Is there any reason to only run it on GET requests?

Not really. At first I thought that we only care about the response body for GET requests, but this was incorrect (e.g. we need a POST request to /api/flairselector to get someone's flair).

**Pull request merged**